src/app/globals.css

```
@@ -1,37 +1,23 @@
-@tailwind utilities;
+@import 'tailwindcss';
 
-*,
-*::before,
-*::after {
-	margin: 0;
-	padding: 0;
-	box-sizing: inherit;
+@plugin '@tailwindcss/typography';
+@plugin 'daisyui' {
+	themes:
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)